geode-core/src/main/java/org/apache/geode/internal/cache/execute/metrics/FunctionServiceStats.java

```diff
@@ -382,22 +382,18 @@ public void startFunctionExecution(boolean haveResult) {
    * Increments the "functionExecutionsCompleted" and "functionExecutionCompleteProcessingTime"
    * stats.
    *
-   * @param start The start of the functionExecution (which is decremented from the current time to
-   *        determine the function Execution processing time).
+   * @param elapsed The elapsed time for function execution in nanoseconds
    * @param haveResult haveResult=true then update the _functionExecutionHasResultRunningId and
    *        _functionExecutionHasResultCompleteProcessingTimeId
    */
-  public void endFunctionExecution(long start, boolean haveResult) {
-    long ts = getTime();
-
+  public void endFunctionExecution(long elapsed, boolean haveResult) {
```

Comment:
Just curious, I think I saw some endFunctionExecution() method invocations that passed in start time.  Is that a different method signature?

Reply:
`FunctionStats.endFunctionExecution()` takes a start time while `FunctionServiceStats.endFunctionExecution()` takes an elapsed time. The FunctionServiceStats method is only called from FunctionStats, so I think the invocations you saw were probably `FunctionStats.endFunctionExecution()`. I can see how this is confusing since they both have the same name and signature.

Reply:
I renamed `FunctionServiceStats.endFunctionExecution()` to `FunctionServiceStats.endFunctionExecutionWithElapsedTime()` to make it more apparent that it takes an elapsed time as an argument.